webpack.config.js

```diff
@@ -271,6 +271,12 @@ module.exports = (env, argv) => {
                         if (f.startsWith(reactSdkSrcDir)) return true;
                         if (f.startsWith(jsSdkSrcDir)) return true;
 
+                        // Some of the syntax in this package is not understood by
+                        // either webpack or our babel setup.
+                        // When we do get to upgrade our current setup, this should
+                        // probably be removed.
+                        if (f.includes("compound-web")) return true;
```

Comment:
Could we be more specific, there might be transitive imports which also contain the text `compound-web`